components/dialog/index.css

```diff
@@ -12,40 +12,28 @@ governing permissions and limitations under the License.
 
 @import "../overlay/index.css";
 
+
+
 .spectrum-Dialog {
   /* The font-size of the fullscreen dialog header */
   --spectrum-dialog-fullscreen-header-text-size: 28px;
-
+  --spectrum-dialog-min-inline-size: 288px;
   --spectrum-dialog-confirm-small-width: 400px;
   --spectrum-dialog-confirm-medium-width: 480px;
   --spectrum-dialog-confirm-large-width: 640px;
   --spectrum-dialog-confirm-max-inline-size: 90vw;
   --spectrum-dialog-confirm-max-height: 90vh;
-
-  --spectrum-dialog-confirm-hero-height: var(
-    --spectrum-global-dimension-size-1600
-  );
-  --spectrum-dialog-confirm-gap-size: var(--spectrum-global-dimension-size-200);
-  --spectrum-dialog-confirm-description-padding: var(
-    --spectrum-global-dimension-size-25
-  );
-  --spectrum-dialog-confirm-description-margin: calc(
-    var(--spectrum-global-dimension-size-25) * -1
-  );
-  --spectrum-dialog-confirm-footer-padding-top: var(
-    --spectrum-global-dimension-static-size-500
-  );
-  --spectrum-dialog-confirm-gap-size: var(--spectrum-global-dimension-size-200);
-  --spectrum-dialog-confirm-buttongroup-padding-top: var(
-    --spectrum-global-dimension-static-size-500
-  );
-  --spectrum-dialog-confirm-close-button-size: var(
-    --spectrum-global-dimension-size-400
-  );
-  --spectrum-dialog-confirm-close-button-padding: calc(
-    26px - var(--spectrum-global-dimension-size-175)
-  );
-  --spectrum-dialog-confirm-divider-height: var(--spectrum-global-dimension-static-size-25);
+  --spectrum-dialog-confirm-padding-300: var(--spectrum-spacing-300);
+  --spectrum-dialog-confirm-padding-200: var(--spectrum-spacing-200);
```

Comment:
Can we rename these back to being for their purpose rather than their value? It looks like they used to be 
`--spectrum-dialog-confirm-divider-margin-top` and 
`--spectrum-dialog-confirm-divider-margin-bottom`, and are still only used for that purpose.

Reply:
Done!